ui/app/components/app/gas-customization/gas-modal-page-container/advanced-tab-content/advanced-tab-content.component.js

```diff
@@ -27,21 +27,22 @@ export default class AdvancedTabContent extends Component {
     isSpeedUp: PropTypes.bool,
     isEthereumNetwork: PropTypes.bool,
     customGasLimitMessage: PropTypes.string,
-    minimumGasLimit: PropTypes.number.isRequired,
+    minimumGasLimit: PropTypes.number,
+    hideAdvancedTimeEstimates: PropTypes.bool,
```

Comment:
This API seems a bit confusing. `hideAdvancedTimeEstimates` hides both the the time estimates and the gas prediction chart, leaving only the gas inputs. Instead of adding a "hide everything except the gas inputs" flag, could you embed the `AdvancedGasInputs` directly in the new modal instead?

Reply:
Done in 15df6a697